trio/_core/_tests/tutil.py

```diff
@@ -43,7 +42,7 @@
     with s:
         try:
             s.bind(("::1", 0))
-        except OSError:
+        except OSError:  # pragma: no cover # since support for 3.7 was removed
```

Comment:
I don't think the `#` between `no cover` and `since support for 3.7 was removed` is necessary, though I actually don't remember this pattern from elsewhere so IDK if trio prefers the more prose-like `# pragma: no cover since support for 3.7 was removed`

Reply:
`mypy` will error out on action comments with trailing content unless separated by `#`. Idk how codecov handles it, but it seems like a reasonable thing to default to differentiating what has an effect and what's a pure comment.